README.md

```diff
@@ -13,26 +13,27 @@ const {
 } = utils;
 ```
 
-* **Remote Control** - The remote control utility requires iframe HTML Element with loaded Jitsi Meet.
+* **Remote Control** - The remote control utility requires iframe HTML Element that will load Jitsi Meet.
+
+Enable the remote control:
 ```Javascript
 const remoteControl = new RemoteControl(iframe/* the Jitsi Meet iframe */);
-...
-...
-...
-remoteControl.dispose();
 ```
-* **Desktop Sharing**
+
+To disable the remote control:
 ```Javascript
-setupScreenSharingForWindow(window/* the Jitsi Meet window object */);
+remoteControl.dispose();
 ```
-If Jitsi Meet is loaded in an iframe HTML element which is stored in the `iframe` variable, the code will look like:
+
+NOTE: `dispose` method will be called automatically when the Jitsi Meet iframe unload.
+
+* **Desktop Sharing**
 ```Javascript
-setupScreenSharingForWindow(iframe.contentWindow);
+setupScreenSharingForWindow(iframe/* the Jitsi Meet iframe */);
```

Comment:
Not really! 

But for the remote control we need the iframe. So it looks more consistent. 

Also IMHO it looks easier for the users to pass just an HTML element instead of having to select the window of the iframe. 

Also in order to make it work properly with iframe reloads you have to listen for iframe's load event and call the setup function every time. Now this is embedded into the function.